src/engine/tools/engine-suite/src/engine_test/integrations.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I think this would have to be an array of objects, I find it confusing to put integration_x, why not just a parent integration and then the objects that represent each one?

```
{       
   "integration": [
             {},
             {},
       ]    
}
```